kubernetes/debug.yml

```diff
@@ -19,6 +19,8 @@ metadata:
     # paste annotations from failing resource here
     # or modify /secretkeys after exec-ing into the pod
     secret/MYSQL_USER: global/global/global/mysql_user
+    # paste PKI annotations here to test PKI resolution
+    # pki/www.my-website.com: pki/issue/my-role?common_name=www.my-website.com
```

Comment:
confirm this is actually a valid annotation key

Reply:
Yeah this isn't a real key, supposed to be more like a syntax example. I can move it somewhere else if there's a better place or delete it.